src/development/tools/devtools/memory.md

```diff
@@ -434,3 +434,11 @@ to show its callees.
 
 The bottom-up view shows the list of different
 call stacks that have allocated the instances.
+
+## Other resources
+
+Check out this step-by-step guided [Memory View Tutorial][memory-tutorial] 
+to learn how you can monitor a Flutter app's memory usage 
+and detect any memory leaks using the Memory View.
+
+[memory-tutorial]: https://medium.com/@fluttergems/mastering-dart-flutter-devtools-memory-view-part-7-of-8-e7f5aaf07e15
```

Comment:
```suggestion
To learn how to monitor an app's memory usage
and detect memory leaks using DevTools,
check out a guided [Memory View tutorial][memory-tutorial].

[memory-tutorial]: {{site.medium}}/@fluttergems/mastering-dart-flutter-devtools-memory-view-part-7-of-8-e7f5aaf07e15
```

Again, can you confirm whether these articles are behind a paywall?

Reply:
Yes, none of these articles are behind the paywall.